src/pages/auth/schoolVerification.jsx

```diff
@@ -3,55 +3,48 @@ import { path } from '@/routes/path';
 import { useEffect, useState } from 'react';
 import { useLocation, useNavigate } from 'react-router-dom';
 import { SkipHeader } from '@/components/join/SkipHeader';
-import { useCheckSchoolStatus } from '@/hooks/use-CheckSchoolStatus';
+import { useCheckSchoolStatus } from '@/hooks/useCheckSchoolStatus';
+import { UNIV_STATUS } from '@/constants/universityStatus';
 
 export const SchoolVerification = () => {
   const location = useLocation();
   const navigate = useNavigate();
-  const university = location.state;
-  const [status, setStatus] = useState('');
+  const { university, isFirst } = location.state;
 
-  const { mutate } = useCheckSchoolStatus();
+  const { data: status } = useCheckSchoolStatus();
 
   useEffect(() => {
-    mutate(undefined, {
-      onSuccess: (data) => {
-        setStatus(data.status);
-      },
-      onError: (error) => {
-        alert(error);
-      },
-    });
-  }, []);
-
-  useEffect(() => {
-    if (status === 'APPROVED' || status === 'PENDING') {
-      navigate(`../../${path.home}`);
+    if (status === UNIV_STATUS.APPROVE || status === UNIV_STATUS.PENDING) {
+      navigate(path.home, { replace: true });
     }
     if (university === undefined) {
-      navigate(`../${path.signup.school}`);
+      navigate(path.signup.base + '/' + path.signup.school, { replace: true });
     }
   }, [status, university]);
 
   return (
-    <div className="flex flex-col w-full h-full">
-      <SkipHeader />
-      <div className="flex flex-col flex-1 gap-10 px-4 py-5">
+    <div className="flex h-full w-full flex-col">
+      <SkipHeader isFirst={isFirst} />
+      <div className="flex flex-1 flex-col gap-10 px-4 py-5">
         <div className="flex flex-col gap-[3.125rem]">
-          <div className="flex justify-center w-full text-pageTitle">
+          <div className="flex w-full justify-center text-pageTitle">
             How will you verify your school?
           </div>
           <div className="flex flex-col gap-[1.875rem]">
             <MainWhiteButton
               onClick={() =>
-                navigate(path.signup.verify.email, { state: university })
+                navigate(path.signup.verify.email, {
+                  state: { university: university, isFirst: isFirst },
+                })
```

Comment:
객체 키와 값으로 할당하려는 변수 이름이 같을 경우 객체 리터럴 축약 문법에 의해 아래처럼 쓸 수 있다고 합니다
```suggestion
                navigate(path.signup.verify.email, {
                  state: { university, isFirst },
                })
```

Reply:
오 좋습니다 반영하겠습니다